src/Facebook/InstantArticles/Client/Helper.php

```diff
@@ -37,6 +37,7 @@ public function __construct($facebook)
      * @param string $appID
      * @param string $appSecret
      *
+     * @return static
```

Comment:
@everton-rosario 

Using `static` instead of `Helper` makes it also easier when extending `Helper`, because then:

```php
class Foo extends Helper
{}

$foo = Foo::create();
``

it will be clear that `$foo` is an instance of `Foo` rather than `Helper`.